### PR TITLE
Preparing for new release

### PR DESCRIPTION
**Pull request by wu-haoze**
- update the license year
- update change log
- update CI

### CODE REVIEW

File: .github/workflows/cibuildwheel.yml [hunk 1]
```diff
@@ -107,7 +108,7 @@ jobs:
 
     environment:
       name: pypi
-      url: https://pypi.org/p/vehicle-lang
```
Comment (MatthewDaggitt) [inline .github/workflows/cibuildwheel.yml:110]: Haha, I wonder how that got in there :laughing:

File: THANKS [hunk 2]
```diff
@@ -1,18 +1,18 @@
 Marabou team thanks the following contributors:
 
-Elazar Cohen, Hebrew University of Jerusalem 
-Omri Cohen, Hebrew University of Jerusalem 
+Elazar Cohen, Hebrew University of Jerusalem
+Omri Cohen, Hebrew University of Jerusalem
+Matthew Daggitt, Heriot-Watt University
```
Comment (MatthewDaggitt) [inline THANKS:5]: I'm now at the University of Western Australia

**Review by MatthewDaggitt — APPROVED** [review]
Otherwise looks good to me

File: .github/workflows/cibuildwheel.yml [hunk 3]
```diff
@@ -107,7 +108,7 @@ jobs:
...
+      url: https://pypi.org/project/maraboupy-NeuralNetworkVerification/
```
Comment (wenkokke) [inline .github/workflows/cibuildwheel.yml:111]: That should just be:
```
https://pypi.org/p/maraboupy
```

**Review by guykatzz — APPROVED** [review]

File: .github/workflows/cibuildwheel.yml (hunk 3 above)
Comment (wu-haoze) [reply .github/workflows/cibuildwheel.yml:111]: Ah thanks. I made the change!

**Pull request merged**